### PR TITLE
feat(wren-ai-service): Implement evaluation metrics

### PR DESCRIPTION
**Pull request by paopa**
This PR aims to enhance support for evaluation metrics by implementing a column-based version, consolidating multiple actual outputs into a single trace, and updating the README to reflect the changes in the evaluation process. Additionally, it includes the removal of outdated evaluation code to streamline the system.

### Screenshots

image

image

image

image

### CODE REVIEW

File: wren-ai-service/eval/README.md
Comment (cyyeh) [inline wren-ai-service/eval/README.md:1]: I am thinking we could rename Wren AI Service to Wren AI LLM Service

File: wren-ai-service/eval/README.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ This document describes the evaluation framework for the Wren AI service. The ev
...
+## Dataset Curation
+
+The dataset curation process is used to prepare the evaluation dataset for the Wren AI service on evaluation purpose. You can follow the steps below to start the curation app:
+
+- copy `.env.example` file to `.env` in the `eval/data_curation` folder and fill in the environment variables
+- execute the command under the `wren-ai-service` folder: `make data_curation_app`
+
 ## Prediction Process
 
 The prediction process is used to produce the results of the Wren AI service. It will create traces and a session on Langfuse to make the results available to the user. You can use the following command to predict the evaluation dataset under the `eval/dataset` directory:
```
Comment (cyyeh) [inline wren-ai-service/eval/README.md:19]: produce the results of the evaluation data using Wren AI LLM Service?

**Review by cyyeh — COMMENTED** [review]
example.py can be deleted?

File: wren-ai-service/eval/README.md
Comment (paopa) [reply wren-ai-service/eval/README.md:1]: discussed offline, and we will propose our thought to members to decide whether rename it.

**Review by cyyeh — APPROVED** [review]
LGTM, thanks for your hard work.

**Pull request merged**